libs/coin-modules/coin-bitcoin/src/types.ts

```diff
@@ -172,3 +173,8 @@ export type BitcoinAccount = Account & { bitcoinResources: BitcoinResources };
 export type BitcoinAccountRaw = AccountRaw & {
   bitcoinResources: BitcoinResourcesRaw;
 };
+
+export type BtcOperationExtra = {
+  inputs?: string[] | undefined;
```

Comment:
```suggestion
  inputs: string[];
```

See my comment above

Reply:
This one is problematic (undoable) as of now as: 

- Old operations have no inputs, this update is not retroactive on old operations as some information (input ids ) available on the original transaction is lost when mapping it to operations here (https://github.com/LedgerHQ/ledger-live/blob/f583161353602588bf98d3de63fe35928d88410a/libs/coin-modules/coin-bitcoin/src/logic.ts#L167). 

Not a big deal imo as fixing the behavior starting in the next release is more crucial, for users that have dangling operations they are still asked to clear cache and resync.